src/lib/mathlib/math/filter/LowPassFilter2p.hpp

```diff
@@ -71,7 +71,7 @@ class LowPassFilter2p
 		_delay_element_1 = {};
 		_delay_element_2 = {};
 
-		_cutoff_freq = math::constrain(cutoff_freq, 5.f, sample_freq / 2); // TODO: min based on actual numerical limit
+		_cutoff_freq = math::max(cutoff_freq, sample_freq * 0.001f);
```

Comment:
Why did you remove the upper bound? That one can still be the Nyquist frequency or a fraction of it.

Reply:
It's already handled above (disabling the filter entirely). https://github.com/PX4/PX4-Autopilot/blob/acf51d54b5d33ab65e78810ad46f3121274d6346/src/lib/mathlib/math/filter/LowPassFilter2p.hpp#L63-L68